### PR TITLE
Fix unit test for region error handling

### LINKED ISSUE #2192: Failed error handling tests for regions
NetworkTest.testErrorHandling fails because of changes made to nupic.core to allow users to register custom python regions.

### PR DESCRIPTION
**Pull request by akhilaananthram**
Fixes #2192.

I changed the expected error messages to match the ones nupic.core actually outputs.

@rhyolight

### CODE REVIEW

**Comment by akhilaananthram** [comment]
@scottpurdy. I made the changes so the tests pass again

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,8 @@ branches:
 
 language: cpp
 
+sudo: required
```
Comment (scottpurdy) [inline .travis.yml]: Why was this necessary?

Comment (akhilaananthram) [inline .travis.yml]: I added that because Travis said: the job is running on container-based infrastructure, which does not allow use of 'sudo', setuid and setguid executables.

This was the solution to getting nupic on my local machine to build on travis.

https://travis-ci.org/akhilaananthram/nupic/jobs/66064665

**Comment by scottpurdy** [comment]
:+1:
